src/providersManager/providerList.ts

```diff
@@ -20,7 +20,7 @@ export const providerList: ProviderList[] = [
   },
   {
     name: 'MASSA_STATION',
-    checkInstalled: isMassaStationInstalled,
+    checkInstalled: isMassaStationAndWalletPluginInstalled,
```

Comment:
we want the wallet plugin to be up

```suggestion
    checkInstalled: isMassaWalletEnabled,
```